staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -134,16 +134,19 @@ func (s *EtcdOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&s.EnableWatchCache, "watch-cache", s.EnableWatchCache,
 		"Enable watch caching in the apiserver")
 
-	fs.IntVar(&s.DefaultWatchCacheSize, "default-watch-cache-size", s.DefaultWatchCacheSize,
+	fs.IntVar(&s.DefaultWatchCacheSize, "default-watch-cache-size", s.DefaultWatchCacheSize, ""+
```

Comment:
Instead of this, let's just switch to using fs.MarkDeprecated()

Reply:
Done, can you PTAL?